6.bulk_Morphology_Elastic_Network/1.train_models/notebooks/1.train_regression_multi_output.ipynb

```diff
@@ -1,419 +1,942 @@
 {
```

Comment:
<p>I don't think this variable is used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>It is used once for the output toml of the model conditions</p>